hack/runtime-migrator/README-rr.md

```diff
@@ -1 +1,178 @@
+```
+[
+  {
+    "runtimeId": "a774bae2-ed8b-464e-85cc-ab8acd4461d5",
+    "shootName": "c-35a9898",
+    "status": "Success",
+    "restoredCRBs": [
+      "admin-cw4mz"
+    ],
+    "restoredOIDCs": [
+      "kyma-oidc-0"
+    ]
+  },
+  {
+    "runtimeId": "exxe4b14-7bc2-4947-931c-f8673793b02f",
+    "shootName": "",
+    "status": "Error",
+    "errorMessage": "Failed to fetch shoot: shoot was deleted or the runtimeID is incorrect"
+  }
+]
+```
+
+In the above example the runtime with the `exxe4b14-7bc2-4947-931c-f8673793b02f` identifier was not found. In such a case verify whether:
+- identifier is correct
+- the corresponding shoot exists, and has `kcp.provisioner.kyma-project.io/runtime-id` label set
+
+The runtime with the `a774bae2-ed8b-464e-85cc-ab8acd4461d5` was successfully restored. The shoot spec was patched, and the following resources recreated:
+- `admin-cw4mz` of type Cluster Role Binding
+- `kyma-oidc-0` of type OpenIDConnect 
+
+#### Runtime must be restored manually
+
+```
+2025/01/13 09:04:56 INFO Starting runtime-restorer
+2025/01/13 09:04:56 gardener-kubeconfig-path: /Users/myuser/Downloads/kubeconfig-garden-kyma-stage.yaml
+2025/01/13 09:04:56 kcp-kubeconfig-path: /Users/myuser/dev/config/sap
+2025/01/13 09:04:56 gardener-project-name: kyma-stage
+2025/01/13 09:04:56 output-path: /tmp
+2025/01/13 09:04:56 dry-run: false
+2025/01/13 09:04:56 input-type: txt
+2025/01/13 09:04:56 input-file-path: /Users/myuser/dev/input/runtime-ids-oidc.txt
+2025/01/13 09:04:56 backup-path: /Users/myuser/backup/results/backup-2025-01-10T13:50:55+01:00
+2025/01/13 09:04:56 restore-crb: true
+2025/01/13 09:04:56 restore-oidc: true
+2025/01/13 09:04:56
+2025/01/13 09:04:56 INFO Reading runtimeIds from input file
+2025/01/13 09:05:01 WARN Verify the current state of the system. Restore should be performed manually, as the backup may overwrite user's changes. runtimeID=a774bae2-ed8b-464e-85cc-ab8acd4461d5
+2025/01/13 09:05:01 ERROR Failed to fetch shoot: shoot was deleted or the runtimeID is incorrect runtimeID=exxe4b14-7bc2-4947-931c-f8673793b02f
+2025/01/13 09:05:01 INFO Restore completed. Successfully restored backups: 0, Failed operations: 1
+2025/01/13 09:05:01 INFO Restore results saved in: /tmp/restore-2025-01-13T09:04:56+01:00/restore-results.json
+```
+
+The `restore-results.json` file contains the following content:
+```
+[
+  {
+    "runtimeId": "a774bae2-ed8b-464e-85cc-ab8acd4461d5",
+    "shootName": "c-35a9898",
+    "status": "UpdateDetected"
+  },
+  {
+    "runtimeId": "exxe4b14-7bc2-4947-931c-f8673793b02f",
+    "shootName": "",
+    "status": "Error",
+    "errorMessage": "Failed to fetch shoot: shoot was deleted or the runtimeID is incorrect"
+  }
+]
+```
+
+The runtime with the `a774bae2-ed8b-464e-85cc-ab8acd4461d5` cannot be automatically restored as there were several updates to the shoot. The runtime must be fixed manually, as there is a risk some updates performed by the user will be overwritten.
```

Comment:
```suggestion
The runtime with the `a774bae2-ed8b-464e-85cc-ab8acd4461d5` cannot be automatically restored, because there were several updates to the shoot. The runtime must be fixed manually, because there is a risk that some updates performed by the user will be overwritten.
```